internal/commands/result.go

```diff
@@ -1216,15 +1224,65 @@ func exportGlSastResults(targetFile string, results *wrappers.ScanResultsCollect
 	defer f.Close()
 	return nil
 }
+
+func exportGlDependencyResults(targetFile string, results *wrappers.ScanResultsCollection, summary *wrappers.ResultSummary) error {
+	log.Println("Creating Gl-sca Report: ", targetFile)
+	var glDependencyResult = new(wrappers.GlDependencyResultsCollection)
+	err := addScanToGlDependencyReport(summary, glDependencyResult)
+	if err != nil {
+		return errors.Wrapf(err, "%s: failed to denerate gl dependency report ", failedListingResults)
```

Comment:
```suggestion
		return errors.Wrapf(err, "%s: failed to denerate gl sca report ", failedListingResults)
```

change the all logs and function names to sca instead of dependency